src/models/daisyworld.jl

```diff
@@ -0,0 +1,163 @@
+using Random
+import StatsBase
+import DrWatson: @dict
```

Comment:
This is the current Travis error. It's just a convenience function, so let's drop this dependency.